### DISCUSSION TITLE
How can I display alpha-nvim horizontally?

### DISCUSSION THREAD

**Discussion by lucaschristiansson** [goolord/alpha-nvim#312] [Q&A] [answered]
I would like to display the dashboard as:

[
Header|Buttons
]

instead of the regular, which is displayed vertically:

[
Header
Buttons
]

Is there an option that would allow this or would I need to customize the alpha-nvim source code to achieve this?

**Comment by goolord** [comment, marked as answer]
it's not presently supported, but it's planned and I've toyed around with doing it myself. you can try implementing it yourself if you'd like, the rendering code is not *that* complicated
